.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
                     python3 -m venv venv
                     . venv/bin/activate
                     pip install requests
-                    python test/run_portability_test.py --test-directories test/smartseq2_single_sample/pr/ test/optimus/pr
+                    python test/run_portability_test.py --test-directories test/smartseq2_single_sample/pr/ test/optimus/pr test/bulk_rna_pipeline/pr
     test_optimus:
         machine: true
 
```

Comment:
You can also add a separate test trigger for the GCP infrastructure. See #204 `.circleci/config.yaml` file changes for example.